### DISCUSSION TITLE
Deta Micro Timeout

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/space-docs#25] [Ideas]
I want to run some code on my micro (i.e. fetching and writing some data) however, this takes some time (i.e. 5 minutes) to download and apply the processing to the initial 300MB to form the final 48MB output.

How can I run this (like the old Deta cloud `deta run` feature) without it violating the timeout of 10s that micro requests have?

**Comment by abdelhai** [comment]
"space run" will come to space soon (will be called "Teletype Actions")
we're working on limits and will share more soon

**Reply by s4m-mo** [reply]
What about tasks that run on startup (e.g. loading a small 40MB dataset) of the Micro? How can I run them without the timeout?
